tutorials/scripting/c_sharp/c_sharp_global_classes.rst

```diff
@@ -21,6 +26,35 @@ The ``MyNode`` type will be registered as a global class with the same name as t
 
 .. image:: img/globalclasses_addnode.webp
 
+The *Select a Node* window for the ``MyNode`` exported property filters the list
+of nodes in the scene to match the assignment restriction.
+
+.. code-block:: csharp
+
+    public partial class Main : Node
+    {
+        [Export]
+        public MyNode MyNode { get; set; }
+    }
+
+
+.. image:: img/globalclasses_exportednode.webp
+
+The inheritance hierarchy is used if a custom type isn't registered.
+
+.. code-block:: csharp
+
+    public partial class MySpecializedNode : MyNode
+    {
+    }
+
+Inspector assignments to an export of type ``MySpecializedNode`` are restricted
+to ``MyNode`` and derivative types. This is because ``MySpecializedNode`` isn't
```

Comment:
For consistency, we seem to generally use "derived type" and not "derivative types".
```suggestion
to ``MyNode`` and derived types. This is because ``MySpecializedNode`` isn't
```